src/concrete/ml/sklearn/glm.py

```diff
@@ -325,7 +325,6 @@ def dump_dict(self) -> Dict:
         # Scikit-Learn
         metadata["alpha"] = self.alpha
         metadata["fit_intercept"] = self.fit_intercept
-        metadata["solver"] = self.solver
```

Comment:
any idea why these are removed ?

Reply:
This comes from the new test we added recently 

https://github.com/zama-ai/concrete-ml/blob/6921c91c6e44e89b662236501f8b5f10bfeeff4a/tests/sklearn/test_sklearn_models.py#L2057

It highlighted a few discrepancies with sklearn models. Here `solver` isn't actually part of this model in sklearn.